projects/plugins/social/composer.json

```diff
@@ -72,6 +72,6 @@
 			"automattic/jetpack-autoloader": true,
 			"automattic/jetpack-composer-plugin": true
 		},
-		"autoloader-suffix": "c4802e05bbcf59fd3b6350e8d3e5482c_socialⓥ1_0_1_alpha"
+		"autoloader-suffix": "c4802e05bbcf59fd3b6350e8d3e5482c_socialⓥ1_1_0_alpha"
```

Comment:
To avoid such unrelated changes, I would recommend that the next time you push a new version of the plugin, you immediately create a new PR that just bumps the version number of the plugin to start a new release cycle.

Reply:
Thanks for the pointer, @jeherve! We will make sure we keep this in mind.

Reply:
Cool, does that mean in future we should bump the minor version number? I'll add that to the notes.